admin/class-yoast-notification.php

```diff
@@ -11,24 +11,30 @@ class Yoast_Notification {
 	/**
 	 * Contains optional arguments:
 	 *
-	 * -  type: The notification type, i.e. 'updated' or 'error'
-	 * -    id: The ID of the notification
-	 * - nonce: Security nonce to use in case of dismissible notice.
+	 * -             type: The notification type, i.e. 'updated' or 'error'
+	 * -               id: The ID of the notification
+	 * -            nonce: Security nonce to use in case of dismissible notice.
+	 * -         priority: From 0 to 1, determines the order of Notifications.
+	 * -    dismissal_key: Option name to save dismissal information in, ID will be used if not supplied.
+	 * -     capabilities: Capabilities that a user must have for this Notification to show.
+	 * - capability_check: How to check capability pass: all or any.
+	 * -  wpseo_page_only: Only display on wpseo page or on every page.
 	 *
-	 * @var array
+	 * @var array Options of this Notification.
 	 */
-	private $options;
+	private $options = array();
 
-	/**
-	 * Contains default values for the optional arguments
-	 *
-	 * @var array
-	 */
+	/** @var array Contains default values for the optional arguments */
 	private $defaults = array(
-		'type'      => 'updated',
-		'id'        => '',
-		'nonce'     => null,
-		'data_json' => array(),
+		'type'             => 'updated',
+		'id'               => '',
+		'nonce'            => null,
+		'priority'         => 0.5,
```

Comment:
I really dislike having a broad range going outwards instead of inwards in subtlety.
Maybe even prefer a 'place before/after notification_id' approach..
@omarreiss could you think about this?